cmd/node/factory/structs.go

```diff
@@ -85,9 +85,6 @@ const (
 	MaxTxsToRequest = 100
 )
 
-// ErrShardGenesisBlockNotExits signals that genesis block not exits
-var ErrShardGenesisBlockNotExits = errors.New("genesis block not exits")
```

Comment:
Might leave this err and use it on line 515. Also. Might change the error string to "genesis block **does** not exi**sts**"

Reply:
The error string has been changed with the suggested one. But not created an error variable because in structs.go all returned errors are created with errors.New().

Reply:
ok